src/Libraries/RevitNodes/Elements/FamilyType.cs

```diff
@@ -221,6 +249,145 @@ public static FamilyType ByName(string name)
             };
         }
 
+        /// <summary>
+        /// Create new Family Instance from Geometry (via SAT import)
+        /// </summary>
+        /// <param name="geometry"></param>
+        /// <param name="name"></param>
+        /// <param name="category"></param>
+        /// <param name="templatePath"></param>
+        /// <param name="isVoid"></param>
```

Comment:
No, my point is that if users have a list of geometries and want to convert them to a list of FamilyType objects, the program should automatically figure out which ones are void and which ones are not void.